README.md

```diff
@@ -125,6 +125,22 @@ const { getDairatsForWilaya } = require('@dzcode-io/leblad');
 console.log(getDairatsForWilaya(3)); //returns list of dairats for wilaya 3
 ```
 
+#### getWilayaByPhoneCode(phoneCode: number, projection?: string[])
+
+Takes a phone code and returns the matching wilaya.
+
+**Arguments**
+
+- `phoneCode: number|string` (**required**) the Wilaya's "phoneCode" or the full phone number
+- `projection: string[]` (optional) Array of Wilaya Object attributes to keep
+
+**Examples**
+
+```javascript
+const { getWilayaByPhoneCode } = require('@dzcode-io/leblad');
+
+console.log(getWilayaByPhoneCode(34)); //will the wilaya object ({name: "Béjaïa"...})
+```
```

Comment:
Please, add a line break after this.  
Aand also I suggest adding another statement(here in the same example block) with the full phone number, it's a very nice thing you added 😄